src/transformers/models/roberta/modeling_roberta.py

```diff
@@ -276,14 +276,21 @@ def forward(
 class RobertaSelfOutput(nn.Module):
     def __init__(self, config):
         super().__init__()
+        self.normalize_before = config.normalize_embeddings
```

Comment:
Code here tries to mimic the `normalize_before` logic from `fairseq`:

https://github.com/pytorch/fairseq/blob/4950c56f461db1646872159b2c470fe57ae72c69/fairseq/modules/transformer_layer.py#L132-L145

Reply:
Module overview:

```
            (output): RobertaSelfOutput(
              (dense): Linear(in_features=2560, out_features=2560, bias=True)
              (LayerNorm): LayerNorm((2560,), eps=1e-05, elementwise_affine=True)
              (dropout): Dropout(p=0.1, inplace=False)
            )
```

Extract from Conversion script to see `fairseq` -> Transformers variable initialization for this part:

```
    # self-attention output
    self_output: BertSelfOutput = layer.attention.output
    assert self_output.dense.weight.shape == roberta_layer.self_attn.out_proj.weight.shape
    self_output.dense.weight = roberta_layer.self_attn.out_proj.weight
    self_output.dense.bias = roberta_layer.self_attn.out_proj.bias
    self_output.LayerNorm.weight = roberta_layer.self_attn_layer_norm.weight
    self_output.LayerNorm.bias = roberta_layer.self_attn_layer_norm.bias
```